examples/profile/main.go

```diff
@@ -144,7 +151,15 @@ func profile(w http.ResponseWriter, r *http.Request) {
 
 	client = &yoti.Client{
 		SdkID: sdkID,
-		Key:   key}
+	}
+	client.Key, err = yoti.LoadPEM(key)
```

Comment:
I was thinking we would still have people initialise the client in the same way as before, and we just have the loading of the pem file into a `key` variable above it. 
```go
rsaPrivateKey, keyErr := yoti.LoadPEM(key)
if keyErr != nil {
	errorPage(w, r.WithContext(context.WithValue(
		r.Context(),
		contextKey("yotiError"),
		fmt.Sprintf("%s", keyErr),
	)))
}

client = &yoti.Client{
	SdkID: sdkID,
	Key:   rsaPrivateKey,
}
```
Then it's more clear that it is a required part of initialisation. If we have it like this, it might allow, or just suggest, that clients can initialise the client without a key.

Reply:
We need to return and check for an error value.

Reply:
Have discussed how we want to handle this offline. Have made those changes.